### ISSUE TITLE
AWS Privileges show error for Lightsail

### ISSUE THREAD

**Issue by tomisaacson** [duo-labs/cloudmapper#212]
If I use the example AWS Privileges then Lightsail shows a yellow exclamation mark in the policy:

> Unrecognized actions:
> GetLoadBalancers
>

**Comment by mxmader** [comment]
I've observed the same. Since I have no LightSail resources deployed, I've simply removed it from the policy document JSON.

**Comment by mxmader** [comment]
Interestingly enough, when I ran:
```
python cloudmapper.py collect --account <account name>
```
I could see where one would reasonably conclude that the noted permission is indeed needed:
```
* Getting lightsail:get-load-balancers info
Making call for account-data/./l<account name>/ap-south-1/lightsail-get-load-balancers.json
ClientError: An error occurred (AccessDeniedException) when calling the GetLoadBalancers operation: User: arn:aws:sts::<account ID>:assumed-role/<IAM role name>/<EC2 instance ID> is not authorized to perform: lightsail:GetLoadBalancers
```

**Comment by mxmader** [comment]
After adding `lighsail::GetLoadBalancers` back into the IAM policy, the following manual check succeeds:

```
(cloudmapper) [ec2-user@ip-xxx cloudmapper]$ aws lightsail get-load-balancers --region us-east-1
{
    "loadBalancers": []
}
```

That said, this seems like both a documentation and web console bug on the AWS side.

**Comment by mxmader** [comment]
Reported this upstream: https://forums.aws.amazon.com/thread.jspa?threadID=293141

**Comment by Justin-W** [comment]
> Interestingly enough, when I ran:
> 
> ```
> python cloudmapper.py collect --account <account name>
> ```
> I could see where one would reasonably conclude that the noted permission is indeed needed:
> 
> ```
> * Getting lightsail:get-load-balancers info
> Making call for account-data/./l<account name>/ap-south-1/lightsail-get-load-balancers.json
> ClientError: An error occurred (AccessDeniedException) when calling the GetLoadBalancers operation: User: arn:aws:sts::<account ID>:assumed-role/<IAM role name>/<EC2 instance ID> is not authorized to perform: lightsail:GetLoadBalancers
> ```

I got the same error (13 times, once for each of 13 different regions) when running the collect command.

**Comment by henry-horangi** [comment]
I got passed this error by giving * to lightsail(47 permissions) and denying all except lightsail:GetInstances(46 permissions). In theory, this should be the same as giving lightsail:GetInstances only, but this gives permission to lightsail:GetLoadBalancers as well. I figured there is an api that is not accessible from the API but still exists.

**Comment by 0xdabbad00** [comment]
Resolved in #349 . The ViewOnlyAccess policy now has various LightSail privileges so I don't need this in my own policy. https://github.com/SummitRoute/aws_managed_policies/blob/master/policies/ViewOnlyAccess
